using-timescaledb/continuous-aggregates.md

```diff
@@ -323,24 +323,19 @@ aggregate, such as views that are built on top of the continuous aggregate view.
 
 
 ### Dropping Data with Continuous Aggregates Enabled [](dropping-data)
-When dropping data in a raw hypertable using the [`drop_chunks` function][api-drop-chunks]
-that has a continuous aggregate created on it, we must specify the `cascade_to_materializations`
-argument to the `drop_chunks` call. A value of `true` will cause the continuous aggregate 
-to drop all data associated with any chunks dropped from the raw hypertable. A value
-of `false` will retain data in the continuous aggregate while dropping only the
-raw data.
-
->:TIP: When dropping data from the raw hypertable while retaining data
-on a continuous aggregate, the `older_than` parameter to `drop_chunks`
-has to be longer than the `timescaledb.ignore_invalidation_older_than`
-parameter on the continuous aggregate. That is because we cannot
-process invalidations on data regions where the raw data has been
-dropped.
-
-
-The same argument must also be supplied to the [`add_retention_policy`
-function][api-add-retention] when creating a data retention policy for a
-hypertable with a continuous aggregate.  
+Note that if any still-refreshing (more recent than start_offset) part of the
```

Comment:
```suggestion
Note that if any still-refreshing (more recent than `start_offset`) part of the
```

Reply:
On a more general note: I think it is useful to introduce the concept of "invalidations" because that makes the rest of the text easier to follow and understand.

Reply:
I do think that invalidations are probably too much of an implementation specific detail here.  Users should be able to think of continuous aggregates as just continually updating within a specified region without needing to think about our behind the scenes bookkeeping.